lib/convert.js

```diff
@@ -124,17 +141,22 @@ var sdk = require('postman-collection'),
 
       // ---- Collection Structure and scaffolding ----
       let specWrapper = {},
-
-        /**
-         We need a trie because the decision of whether or not a node
-        is a folder or request can only be made once the whole trie is generated
-        */
+        folderObj, folderStrategy;
+      specWrapper.spec = openapi;
+
+      /**
+       We need a trie because the decision of whether or not a node
+      is a folder or request can only be made once the whole trie is generated
+      */
+      folderStrategy = options.folderStrategy;
+      if (folderStrategy === 'tag') {
+        folderObj = util.generateFolderObjectFromTags(openapi);
+        specWrapper.tags = folderObj;
```

Comment:
@abhijitkane As we are not creating tree in case of tags, so not to confuse someone who reads the code that we are not creating tree in case of tags strategy.